ipv-core/src/main/java/com/ibm/whc/deid/providers/masking/fhir/MaskingProviderBuilder.java

```diff
@@ -122,11 +117,11 @@ public MaskingProviderBuilder(String schemaType,
    * @param tenantId
    * @return
    */
-  private List<FHIRResourceMaskingAction> buildMaskingActions(
+  private ArrayList<FHIRResourceMaskingAction> buildMaskingActions(

```

Comment:
Question: Why change from an interface (List) to an implementation(ArrayList) when defining the return type?